client/components/App.jsx

```diff
@@ -4,12 +4,12 @@ import PriceAndPromoContainer from "./PriceAndPromoContainer.jsx";
 import PriceImage from "./PriceImage.jsx"
 import axios from 'axios';
 
-const AppWrapper = styled.div`
-  width: 370.5px;
-  height: 431.54px;
-`;
 const AppStyled = styled.div`
-  background-color: #1b1e1b;
+  font-family: 'Lucida Grande', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
Have you found this to be a good substitue for Sofia Pro?  that would be great!

Reply:
I have, haven't had any issues so far. I think it looks pretty good.